src/components/Avatar.js

```diff
@@ -117,4 +129,6 @@ function Avatar(props) {
 }
 Avatar.defaultProps = defaultProps;
 Avatar.propTypes = propTypes;
-export default Avatar;
+export default compose(
+    withNetwork(),
+)(Avatar);
```

Comment:
Don't use `compose` when we have only HOC. Regardless let's use a hook instead.

In hooks/ write a new hook useNetwork:

```js
import {useContext} from 'react';
import {NetworkContext} from '../components/OnyxProvider';

export default function useNetwork() {
    return useContext(NetworkContext);
}
```